docs/part-3-serve-and-deploy-the-model/chapter-13-deploy-and-access-the-model-on-kubernetes-with-bentoml.md

```diff
@@ -798,28 +726,23 @@ git status
 
 The output should look like this.
 
-```
+```text
 On branch main
 Your branch is up to date with 'origin/main'.
 
 Changes to be committed:
 (use "git restore --staged <file>..." to unstage)
-    modified:   .mlem.yaml
-    modified:   requirements-freeze.txt
-    modified:   requirements.txt
-    new file:   service_classifier.mlem
+    new file:   kubernetes/deployment.yaml
+    new file:   kubernetes/service.yaml
```

Comment:
At this point, the `bentofile.yaml` isn't commited yet either.

The aactual output is:
```
On branch main
Your branch is up to date with 'origin/main'.

Changes to be committed:
  (use "git restore --staged <file>..." to unstage)
        new file:   kubernetes/deployment.yaml
        new file:   kubernetes/service.yaml
        new file:   src/bentofile.yaml
``` 

It might be relevant to commit it before the k8 deployment step (as to keep it separate from the k8s stuff). I think that would be my suggestion.

Reply:
I agree, I missed that part.